Packs/CVE_2025_31324/Playbooks/playbook-CVE-2025-31324_-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2835 @@
+            args:
+              empty_values: {}
+              remove_keys:
+                value:
+                  simple: "true"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4225,
+          "y": -839.5
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "113":
+    id: "113"
+    taskid: 122907ce-f61f-4d59-8740-68fe7069fcfa
+    type: title
+    task:
+      id: 122907ce-f61f-4d59-8740-68fe7069fcfa
+      version: -1
+      name: Tag and Link Indicators
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "114"
+      - "115"
+      - "119"
+      - "118"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4225,
+          "y": -692.5
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "114":
+    id: "114"
+    taskid: c14bdae0-cb64-432b-860f-1bbf914699e7
+    type: regular
+    task:
+      id: c14bdae0-cb64-432b-860f-1bbf914699e7
+      version: -1
+      name: Tag IP indicators
+      description: commands.local.cmd.new.indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "117"
+    scriptarguments:
+      retry-count:
+        simple: "3"
+      retry-interval:
+        simple: "2"
+      tags:
+        simple: CVE-2025-31324
+      type:
+        simple: IP
+      value:
+        complex:
+          root: ExtractedIndicators.IP
+          filters:
+          - - operator: isNotEmpty
+              left:
+                value:
+                  simple: ExtractedIndicators.IP
+                iscontext: true
+          transformers:
+          - operator: uniq
+      verdict:
+        simple: Malicious
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3637.25,
+          "y": -553.5
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "115":
+    id: "115"
+    taskid: 830bb98c-ec2a-4fb5-840d-fc37a5590c7f
+    type: regular
+    task:
+      id: 830bb98c-ec2a-4fb5-840d-fc37a5590c7f
+      version: -1
+      name: Tag CVE indicators
+      description: commands.local.cmd.new.indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "117"
+    scriptarguments:
```

Comment:
Should we set it to be "Suspicious" or is it created by default with a score of 2?